app/database/models/users.py

```diff
@@ -0,0 +1,50 @@
+from app.database.sqlalchemy_extension import db
+
+
+
+class UserModel(db.Model):
+
+    # Specifying table
+    __tablename__ = "users"
+    __table_args__ = {"extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(30))
+    username = db.Column(db.String(30))
+    email = db.Column(db.String(30))
```

Comment:
is it sufficient (email = 30 letters)? My email address is 28 letters long 😅. I think 100 would be good

Reply:
nice catch!

Reply:
@vatsalkul done